Source/Components/Touchpoints/LiveImages/ImageAnimationManager.swift

```diff
@@ -7,49 +7,47 @@
 
 import Foundation
 
+enum MLBusinessLiveImagesState {
+    case init_multimedia
+    case playing
+    case paused
+    case ready_to_play
+    case download
+    case download_failed
+    case download_success
+    case blocked
```

Comment:
Podríamos cambiar esto para seguir el estilo que propone arqui con respecto a los [nombres](https://github.com/mercadolibre/fury_swift#use-camel-case). Ahí también hay algunos ejemplos de enums que usan lowerCamelCase
```suggestion
    case initMultimedia
    case playing
    case paused
    case readyToPlay
    case download
    case downloadFailed
    case downloadSuccess
    case blocked
```

Reply:
buenisimo! corregido